readinglist/views/article.py

```diff
@@ -14,14 +14,29 @@ def preparer(self, appstruct):
         return strip_whitespace(appstruct)
 
 
+class URL(SchemaNode):
+    """String representing a URL."""
+    schema_type = String
+    validator = colander.All(colander.url, colander.Length(min=1, max=2048))
```

Comment:
Changing the value might imply running some kind of migration depending on the backend. Should we allow that from settings?

Using a constant in code base seems enough IHMO

Putting configuration in schemas is painful :)